### ISSUE TITLE
Use helm to improve F5 CIS upgrades (retired)

### ISSUE THREAD

**Issue by wmhutchison** [BCDevOps/developer-experience#5239]
**Describe the issue**
While OCP 4.14 upgrades were blocked due to Trident, some time was spent investigating the use of helm for managing F5 CIS upgrades instead of the current method of diffing the repo versions and hope the vendor hasn't moved things around. This did not get implemented before Trident upgrade was unblocked, thus am adding this as a Backlog item to complete.

Ideally we would want to use helm to create the manifest content directly, tweak slightly for an ansible template, and PR as needed. If that isn't viable, then helm manifest generation could be used as a stable alternative to seeing what differences are in the manifests, thus allowing PR changes to be quicker and more consistent versus manually investigating github repo content where the vendor is known to mess around with where they place the content we care about.

**Additional context**
Add any other context, attachments or screenshots

**How does this benefit the users of our platform?**
Quicker and more consistent upgrades for software means more time spent on other things.

**Definition of done**
- [ ] Determine if we will use helm to generate manifests directly or as a guide instead.
- [ ] Create new PR for documentation changes reflecting the method by which we will use helm for F5 CIS upgrades.

**Comment by wmhutchison** [comment]
Will be looking into this while the next wave of R&D for OCP upgrades (4.16) takes place. Based on prior investigation, helm charts for f5cis will not likely be used for active manifest content since previous investigation revealed a lack of proper versioning of helm charts to actual F5 CIS versions, only updating helm chart versioning when manifest content needed to change (CRD mods), not actually aligning to speciifc software releases which we get with helm for ako.

**Comment by wmhutchison** [comment]
Discussed with team, closing this since the vendor isn't doing proper versioning for their helm charts unlike what happens for ako.
